lib/valueflow.cpp

```diff
@@ -1658,8 +1658,21 @@ static bool evalAssignment(ValueFlow::Value &lhsValue, const std::string &assign
     return true;
 }
 
+template<class T>
+struct Range {
```

Comment:
`Range` is a pretty generic name. It would be better if this was named `SingleRange` or something like that.

Reply:
Done